pandas/tests/groupby/test_groupby.py

```diff
@@ -134,6 +134,23 @@ def func(dataf):
     result = df.groupby("X", squeeze=False).count()
     assert isinstance(result, DataFrame)
 
+    # https://github.com/pandas-dev/pandas/issues/28330
+    # Test groupby operations on subclassed dataframes/series
+
+    cdf = tm.SubclassedDataFrame(
+        [
+            {"val1": 1, "val2": 20},
+            {"val1": 1, "val2": 19},
```

Comment:
also pls parameterize over as many functions that you can here

Reply:
You can use `reduction_func` from conftest. Probably worth adding something similar for transforming functions as well